lib/src/builder/parsing/members/props_and_state_util.dart

```diff
@@ -62,6 +64,18 @@ annotations.TypedMap getPropsOrStateAnnotation(bool isProps, AnnotatedNode node)
           // ignore: deprecated_member_use_from_same_package
           InstantiatedMeta.fromNode<annotations.StateMixin>(node));
 
+  if (meta != null && meta.potentiallyIncompleteValue is annotations.Props) {
+    // fixme error when this case isn't well formed
```

Comment:
@greglittlefield-wf was there a certain case you had in mind to throw an error for here?

Reply:
Yeah I was thinking if we couldn't parse the value, then we should throw, but it looks like that's probably already handled by accessing `meta?.value`... So maybe this was just a bad FIXME